app/assets/javascripts/facility_journal.js

```diff
@@ -0,0 +1,55 @@
+document.addEventListener("DOMContentLoaded", function() {
+  $(function(){
```

Comment:
I think this generic function wrapped in jQuery (`$(function(){`) is no longer needed now that we are in the context of `"DOMContentLoaded"`.  Can you try removing this line and line 13 and seeing if things still behave?